File: Logic Building/2_LOOPING & PATTERNS/4_Pattern Printing/13/13.go
// Q13: Print consecutive numbers pattern (1, 23, 456, 78910).
// Input: An integer n
// Output: Continuous number pattern

package main

import "fmt"

func main() {
    var n int
    fmt.Scan(&n)
    num := 1
    for i := 1; i <= n; i++ {
        row := ""
        for j := 0; j < i; j++ {
            row += fmt.Sprintf("%d", num)
            num++
        }
        fmt.Println(row)
    }
}

File: Logic Building/2_LOOPING & PATTERNS/4_Pattern Printing/13/13.java
// Q13: Print consecutive numbers pattern (1, 23, 456, 78910).
// Input: An integer n
// Output: Continuous number pattern

import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        int num = 1;
        for (int i = 1; i <= n; i++) {
            for (int j = 0; j < i; j++) {
                System.out.print(num);
                num++;
            }
            System.out.println();
        }
    }
}
